.github/workflows/bazel_build.yml

```diff
@@ -0,0 +1,21 @@
+name: Bazel Build
+run-name: ${{ github.actor }} is building with bazel
+on: [push]
+jobs:
+  bazel-build:
```

Comment:
@golovasteek Github Actions default to come with bazelisk, which I think you should be using to build. See for example https://github.com/Qwello/zvt/blob/main/.github/workflows/bazel.yml, which does not do caching though.

Reply:
Basically `bazel` and `bazelisk` are the same in the github actions.

I see that when running bazel it downloads the right version:

```
2023/11/23 23:48:53 Downloading https://releases.bazel.build/6.4.0/release/bazel-6.4.0-linux-x86_64...
```